03_json_storage/src/models/json.types.ts

```diff
@@ -0,0 +1,2 @@
+export type Data = string | number | boolean | null | Data[];
+export type Json = Record<string, Data> | Data[];
```

Comment:
I think this names is bad.

Reply:
You're right, so I renamed those types like this:
```typescript
export type JsonDataValue = string | number | boolean | null | JsonDataValue[];
export type JsonObject = Record<string, JsonDataValue> | JsonDataValue[];
```